ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -114,47 +127,45 @@ public void onInitializeSuccess() {
           @Override
           public void onInitializeError(String errorMessage) {
             Log.d(TAG, "SDK init failed: " + VungleBannerAdapter.this);
+            VungleListener listener = getVungleListener();
             mVungleManager.removeActiveBannerAd(mPlacementId);
-            if (mPendingRequestBanner && mVungleListener != null) {
-              mVungleListener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
+            if (mPendingRequestBanner && listener != null) {
+              listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
             }
           }
         });
   }
 
   void destroy(@Nullable View adView) {
     Log.d(TAG, "Vungle banner adapter try to destroy:" + this);
-    if (adView == mAdLayout) {
+    if (adView == mAdLayout.get()) {
```

Comment:
it's for destroy() below, if we never set mAdLayout, but create banner somehow it should be finished